### PR TITLE
feat: update assingment and reminder emails

### PR DESCRIPTION
**Pull request by bbaker6225**
ENT-3812

### CODE REVIEW

File: src/components/LicenseRemindModal/emailTemplate.js [hunk 1]
```diff
@@ -1,15 +1,12 @@
...
-  greeting: 'Your Learning Manager has provided you with a license to give you access to take a course(s) on edX.',
-  body: `You have been assigned a license to an edX Enterprise Subscription. edX offers high-quality online courses from the world's best universities and institutions. After activating your license, you'll have instant access to enroll in a wide-array of courses.
-
-Activate your license and begin your learning journey: {LICENSE_ACTIVATION_LINK}
+import { getSubscriptionContactText } from '../../utils';
 
-edX Login: {USER_EMAIL}
-Expiration Date: {EXPIRATION_DATE}
+const emailTemplate = {
+  greeting: 'We noticed you haven’t had a chance to start learning on edX!  It’s easy to get started and browse the course catalog.',
```
Comment (iloveagent57) [inline src/components/LicenseRemindModal/emailTemplate.js:4]: nit: that's a curly quote in "haven't", not sure if intentional.

File: src/components/LicenseRemindModal/emailTemplate.js [hunk 2]
```diff
@@ -1,15 +1,12 @@
...
+  body: `{ENTERPRISE_NAME} partnered with edX to give everyone access to high-quality online courses. Start your subscription and browse courses in nearly every subject from Data Analytics, Digital Media, Business & Leadership, Communications, Computer Science and so much more. Courses are taught by experts from the world’s leading universities and corporations.
```
Comment (iloveagent57) [inline src/components/LicenseRemindModal/emailTemplate.js:5]: Language nit: "...and browse courses in nearly every subject from Data Analytics, Digital Media, Business & Leadership, Communications, Computer Science and so much more" - this feels like it should have a "to" somewhere in the sentence, for example "...browse courses in nearly every subject, from Analytics to Zoology".  Consider changing to "...browse courses in nearly every subject, including [the list of your subjects] and so much more."

**Review by iloveagent57 — COMMENTED** [review]
Top-notch code, minor suggestions about words.

File: src/components/LicenseRemindModal/emailTemplate.js (hunk 2 above)
Comment (bbaker6225) [reply src/components/LicenseRemindModal/emailTemplate.js:5]: good suggestion, I'll pass it on to Joe

**Comment by bbaker6225** [comment]
@iloveagent57 updated with your suggestions

**Review by iloveagent57 — APPROVED** [review]

**Pull request merged**